docs/src/chaos/periodicity.md

```diff
@@ -145,3 +145,6 @@ estimate_period(v, :periodogram, t)
 ```@example MAIN
 estimate_period(v, :zerocrossing, t)
 ```
+```@example MAIN
+estimate_period(v, :yin, t, sr=Int64(1/Δt), f0_min=0.01, f0_max=0.2, f_step=500, w_len=2000, harmonic_threshold=0.1)
```

Comment:
Why are all these keywords necessary? The method doesn't perform well for default keywords? Then perhaps we should rethink the value of the default keywords?

Reply:
Fair point. They can matter if `Δt` is changed. For 0.1, however, the default values are ok.
It really only needs `f0_min` to be adjusted to close to 0, since the frequency is ~0.01. The default is 100, which is reasonable for several audio signals, but definitely not this case. 
Adjusting `f0_max` should speed the calculation up, but it doesn't make much of a difference.